composer.json

```diff
@@ -238,6 +238,9 @@
             },
             "drupal/video_embed_field": {
                 "Issue #3215817: Presentational attributes such as 'border', 'align', or 'bgcolor' are used. CSS should be used for styling instead.": "https://git.drupalcode.org/project/video_embed_field/-/merge_requests/3.diff"
+            },
+            "drupal/easy_breadcrumb": {
```

Comment:
Not sure why this is showing as added, but i think it should be there as per https://github.com/ymcatwincities/openy/commit/44a6074b392c06fe88093bb0f1dab5537c787648